### PR TITLE
Tag editor: Use darker shadow for toolbar to improve button visibility on white cover art

### PR DESCRIPTION
**Pull request by soncaokim**

### CODE REVIEW

**Comment by Octoton** [comment]
Personally, I find this to jarring

Before:
before

After:
after

**Comment by soncaokim** [comment]
Wait a sec, the change concerns tag editor screen, not the now playing screen (as per your screenshots)

**Comment by soncaokim** [comment]
Here under, the before (left) vs after (right), where it is easier to see the back button

image

**Comment by Octoton** [comment]
> Wait a sec, the change concerns tag editor screen, not the now playing screen (as per your screenshots)

My bad, I mistook the place because both shadow use seem to overkill to me (didn't see the move to a stronger shadow in player before this 😅). 
Clearly the existing one is not enough to see correctly the button but the strong one seems too much. Perhaps if it was longer it would be little better, but if there exist a middle ground between normal and strong it would seems less jarring

**Review by AdrienPoupa — APPROVED** [review]

**Comment by soncaokim** [comment]
> Clearly the existing one is not enough to see correctly the button but the strong one seems too much.

@Octoton should be better after this change: https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/682/commits/59740f5a6389c7828d729037141e675a1328c5d5

File: app/src/main/res/layout/activity_album_tag_editor.xml [hunk 1]
```diff
@@ -139,7 +139,7 @@
     <View
         android:layout_width="match_parent"
         android:layout_height="?android:attr/actionBarSize"
```
Comment (Octoton) [inline app/src/main/res/layout/activity_album_tag_editor.xml:141]: could you replace the height with:
android:layout_height="@dimen/toolbar_scrim_height"

this is already used in the flat player and to me it seems like the best of both word, dark enough with the fifty_percent_black_overlay and big enough to not be too noticeable.

**Review by Octoton — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_album_tag_editor.xml (hunk 1 above)
Comment (soncaokim) [reply app/src/main/res/layout/activity_album_tag_editor.xml]: Done

**Review by Octoton — APPROVED** [review]

**Pull request merged**